README.md

```diff
@@ -144,6 +144,7 @@ These derived images include a set of standard capabilities that enable many of
 - Node.js 8.10.0 LTS
 - Powershell Core v6.0.1
 - Python 2.7.12 and 3.5.2
+- Ruby 2.3.1
```

Comment:
Why did we choose this version? Aren't there more versions of Ruby we want on these images? Should we install or add to tools cache?

Reply:
This is the one that is installed by apt-get.  There is supposedly a way to install different versions, but I have not found it to be successful.  I think the tools cache mechanism is probably the better (and more stable) way to install multiple ruby versions.

Reply:
One thing to watch out for with apt-get is that you might get different versions on 14.04 vs. 16.04. Indeed for Ruby, I'm seeing this:

- Ubuntu 14.04 gets ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
- Ubuntu 16.04 gets ruby 2.3.1p112 (2016-04-26) [x86_64-linux-gnu]

So we should update the README like "Ruby 1.9.3 (14.04) or 2.3.1 (16.04)".

Reply:
ah!  Thanks for letting me know!